platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -60,6 +60,7 @@ RUN pip install crontab
 # Install dependencies for Dynamic Port Breakout
 RUN pip install xmltodict==0.12.0
 RUN pip install jsondiff==1.2.0
+RUN pip install ijson
```

Comment:
Can you please pin down the version of this package? In the slave dockers it appears to be pinned as `ijson==2.6.1`

Reply:
Modified as requested.